### DISCUSSION TITLE
Duda

### DISCUSSION THREAD

**Discussion by JorgePadronO** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#236] [Q&A]
Buenas noches compañer@s. alguien puede decirme si se quedo el trabajo de html para mañana? o se dio más tiempo, lo que pasa es que he estado ocupado con cosas de la escuela y apenas haré mi ejercicio (veré las sesiones apenas)

**Comment by AndresAVargas** [comment]
Hoy el MC comento que la practica de HTML la podríamos entregar hasta el domingo

**Reply by RocioGH** [reply]
Que bien, gracias por avisar, me pasa igual pero con mi trabajo :')

**Comment by JuanSalvador73** [comment]
Las entregas se pueden hacer hasta el domingo 6. No han mencionado hasta a que hora, supongo que antes de la medianoche.
